packages/google-pay-integration/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "root": "packages/google-pay-integration",
+  "sourceRoot": "packages/google-pay-integration/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/google-pay-integration/**/*.{ts,tsx}"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": [
+        "coverage/packages/google-pay-integration"
+      ],
+      "options": {
+        "jestConfig": "packages/google-pay-integration/jest.config.js"
+      }
+    }
+  },
+  "tags": [
+    "scope:integration",
+    "scope:shared"
```

Comment:
Same question, just want to confirm the tag usage.

Reply:
Do we need to add multiple tags here?

Reply:
No, not for `google-pay-integration`, this is not intentional. But for `checkout-button-integration`, it is intentional because I want `checkout-button-integration` to be importable by `google-pay-integration` so it can be extended.

Reply:
Yes, it makes sense for checkout-button-integration :D